packages/react-components/react-utilities/src/hooks/useOnClickOutside.cy.tsx

```diff
@@ -90,4 +128,53 @@ describe('useOnClickOutside', () => {
         expect(onOutsideClick).to.be.calledTwice;
       });
   });
+
+  describe('iframes', () => {
+    beforeEach(() => {
+      cy.clock(new Date(), ['setInterval']);
+    });
+
+    it('is invoked on a frame focus (disabledFocusOnIframe={false})', () => {
+      const onOutsideClick = cy.spy();
+
+      mount(<IFrameExample disabledFocusOnIframe={false} onOutsideClick={onOutsideClick} />);
```

Comment:
nit: I would also test the example without explicitly passing the `disabledFocusOnIframe` prop to test the default value behavior.

Reply:
This is covered by unit tests that are already in place 🐱